File: Frontend/model/Customer.js
export class Customer{
    constructor(id,name,address,mobile,salary) {
        this.id=id;
        this.name=name;
        this.address=address;
        this.mobile=mobile;
        this.salary=salary;
    }

    get id() {
        return this._id;
    }

    set id(value) {
        this._id = value;
    }

    get name() {
        return this._name;
    }

    set name(value) {
        this._name = value;
    }

    get address() {
        return this._address;
    }

    set address(value) {
        this._address = value;
    }

    get mobile() {
        return this._mobile;
    }

    set mobile(value) {
        this._mobile = value;
    }

    get salary() {
        return this._salary;
    }

    set salary(value) {
        this._salary = value;
    }
}

File: Backend/src/main/java/com/sudeera/backend/entity/Customer.java
package com.sudeera.backend.entity;

import com.sudeera.backend.dao.util.SuperEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;


@Entity
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Data
@Table(name = "customer")
public class Customer extends SuperEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    @Column(name="name")
    private String name;
    @Column(name = "address")
    private String address;
    @Column(name = "mobile")
    private String mobile;
    @Column(name = "salary")
    private double salary;
    @OneToMany(mappedBy = "customer",targetEntity = Orders.class)
    private List<Orders> ordersList=new ArrayList<>();

    public Customer(int id) {
        this.id = id;
    }
}
